.circleci/config.yml

```diff
@@ -112,6 +115,8 @@ jobs:
       - run:
           name: Test coverage
           command: go test -cover ./cmds/... ./pkg/...
+      - codecov/upload:
+          file: {{ coverage_report_filepath }}
```

Comment:
It looks like yaml has a parse error here. I don't know yaml enough to help. Perhaps look use this snippet from fiano? https://github.com/linuxboot/fiano/blob/master/.circleci/config.yml#L108

Reply:
yeah, I tried to do as suggested on the codecov docs but it yields this error :( I'll go with the curl command instead